src/manifolds/Euclidean.jl

```diff
@@ -54,125 +54,125 @@ Compute the Euclidean distance between two points on the [`Euclidean`](@ref)
-    vecs = [_euclidean_basis_vector(x, i) for i in eachindex(x)]
+function get_basis(M::Euclidean{<:Tuple,ℂ}, p, B::ArbitraryOrthonormalBasis)
+    vecs = [_euclidean_basis_vector(p, i) for i in eachindex(p)]
     return PrecomputedOrthonormalBasis([vecs; im * vecs])
 end
-function get_basis(M::Euclidean, x, B::DiagonalizingOrthonormalBasis)
-    vecs = get_basis(M, x, ArbitraryOrthonormalBasis()).vectors
-    kappas = zeros(real(eltype(x)), manifold_dimension(M))
+function get_basis(M::Euclidean, p, B::DiagonalizingOrthonormalBasis)
+    vecs = get_basis(M, p, ArbitraryOrthonormalBasis()).vectors
+    kappas = zeros(real(eltype(p)), manifold_dimension(M))
     return PrecomputedDiagonalizingOrthonormalBasis(vecs, kappas)
 end
 
-function get_coordinates(M::Euclidean{<:Tuple,ℝ}, x, v, B::ArbitraryOrDiagonalizingBasis)
+function get_coordinates(M::Euclidean{<:Tuple,ℝ}, p, X, B::ArbitraryOrDiagonalizingBasis)
     S = representation_size(M)
     PS = prod(S)
-    return reshape(v, PS)
+    return reshape(X, PS)
 end
-function get_coordinates(M::Euclidean{<:Tuple,ℂ}, x, v, B::ArbitraryOrDiagonalizingBasis)
+function get_coordinates(M::Euclidean{<:Tuple,ℂ}, p, X, B::ArbitraryOrDiagonalizingBasis)
     S = representation_size(M)
     PS = prod(S)
-    return [reshape(real(v), PS); reshape(imag(v), PS)]
+    return [reshape(real(X), PS); reshape(imag(X), PS)]
 end
 
-function get_vector(M::Euclidean{<:Tuple,ℝ}, x, v, B::ArbitraryOrDiagonalizingBasis)
+function get_vector(M::Euclidean{<:Tuple,ℝ}, p, X, B::ArbitraryOrDiagonalizingBasis)
     S = representation_size(M)
-    return reshape(v, S)
+    return reshape(X, S)
 end
-function get_vector(M::Euclidean{<:Tuple,ℂ}, x, v, B::ArbitraryOrDiagonalizingBasis)
+function get_vector(M::Euclidean{<:Tuple,ℂ}, p, X, B::ArbitraryOrDiagonalizingBasis)
     S = representation_size(M)
-    N = div(length(v), 2)
-    return reshape(v[1:N] + im * v[N+1:end], S)
+    N = div(length(X), 2)
+    return reshape(X[1:N] + im * X[N+1:end], S)
 end
 
-function hat(M::Euclidean{N,ℝ}, x, vⁱ) where {N}
+function hat(M::Euclidean{N,ℝ}, p, vⁱ) where {N}
     return reshape(vⁱ, representation_size(TangentBundleFibers(M)))
 end
 
-hat!(::Euclidean{N,ℝ}, v, x, vⁱ) where {N} = copyto!(v, vⁱ)
+hat!(::Euclidean{N,ℝ}, v, p, vⁱ) where {N} = copyto!(v, vⁱ)
 
-@doc doc"""
+@doc raw"""
     injectivity_radius(M::Euclidean)
 
-Return the injectivity radius on the [`Euclidean`](@ref) `M`, which is $\infty$.
+Return the injectivity radius on the [`Euclidean`](@ref) `M`, which is $∞$.
 """
 injectivity_radius(::Euclidean) = Inf
 
-@doc doc"""
-    inner(M::Euclidean, x, v, w)
+@doc raw"""
+    inner(M::Euclidean, p, X, Y)
 
 Compute the inner product on the [`Euclidean`](@ref) `M`, which is just
 the inner product on the real-valued or complex valued vector space
-of arrays (or tensors) of size $n_1\times n_2 \times \cdots \times n_i$, i.e.
+of arrays (or tensors) of size $n_1 \times n_2  \times  …  \times n_i$, i.e.
```

Comment:
Would `×` have the wrong spacing here?

Reply:
No, replaced them, too. I somehow missed too often to replace `\times`. Have done that with a global replace now.